lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/statemachine/events/ExitNestedJourneyEvent.java

```diff
@@ -8,6 +8,7 @@
 
 import java.util.Map;
 
+@SuppressWarnings({"java:S7027"}) // Circular dependency with Event interface
```

Comment:
:eyes: just confirming my thinking that we'd be better off implementing this in TS!